Templates/Ros2RoboticManipulationTemplate/Template/Examples/panda_moveit_config_demo.launch.py

```diff
@@ -0,0 +1,119 @@
+import os
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument, OpaqueFunction
+from launch.substitutions import LaunchConfiguration, PathJoinSubstitution
+from launch.conditions import IfCondition, UnlessCondition
+from launch_ros.actions import Node
+from launch_ros.substitutions import FindPackageShare
+from launch.actions import ExecuteProcess
+from ament_index_python.packages import get_package_share_directory
+from moveit_configs_utils import MoveItConfigsBuilder
+
+from launch_ros.actions import ComposableNodeContainer
+from launch_ros.descriptions import ComposableNode
+
+
+def generate_launch_description():
+
+    declared_arguments = []
+
+    return LaunchDescription(
+        declared_arguments + [OpaqueFunction(function=launch_setup)]
+    )
+
+
+def launch_setup(context, *args, **kwargs):
+
+    current_file_path = os.path.abspath(__file__)
```

Comment:
Nitpick: use `pathlib` for handling paths.

Reply:
Thanks Piotr for your review. Since it is nitpicking, I've decided to move it to next PR. Let us merge this one.